src/components/AnimateSharedLayout/utils/crossfader.ts

```diff
@@ -71,42 +70,47 @@ export function createCrossfader(): Crossfader {
         finalCrossfadeFrame = null
         let hasUpdated = false
 
```

Comment:
Everything below is mostly just whitespace changes for the Promise return. Only new line is the `resolve()` from the `onComplete()`.